docs/5.0/preview/cloud.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Teleport Cloud Preview
+description: Teleport hosted and managed by Teleport team.
+---
+
+# Teleport Cloud Preview
+
+Teleport cloud is a managed an hosted service of Teleport.
+Request access to preview [here](https://goteleport.com/get-started/).
+
+## Status
+
+Teleport cloud is currently only recommended for environments that do not require
+meeting strict compliance standards or SLA uptime guarantees.
+
+We believe that Teleport cloud environment is secure, however we scrutinizing
+and executing on our security roadmap and working with independent security
+auditors to identify any gaps.
+
+## Roadmap
+
+For the first half of 2021, we will be working with auditors to realize the
+threat modeling and security controls.
+
+Once we realize, test and audit the security model we will start working towards meeting
+compliance programs such as FedRamp.
+
+## Billing
+
+Usage will be collected, but billing will not be set up until February.
+Preview version will be free of charge to preview customers until
+we commit to SLA in February.
+
+## Security
```

Comment:
I still don't believe this section really outlines the security model, and would be better not to include until we can go through in sufficient detail.

Reply:
ok move to FAQ.